products/rhel10/profiles/cis.profile

```diff
@@ -0,0 +1,17 @@
+documentation_complete: true
+
+metadata:
+    SMEs:
+        - marcusburghardt
+
+reference: https://www.cisecurity.org/benchmark/red_hat_linux/
+
+title: 'DRAFT - CIS Red Hat Enterprise Linux 10 Benchmark for Level 2 - Server'
+
+description: |-
+    This is a draft profile for experimental purposes.
+    It is based on the CIS RHEL9 profile, because an equivalent policy for RHEL10 didn't yet exist
```

Comment:
Use a space between RHEL and 9 and RHEL and 10.

Reply:
Fixed.